src/legacy/format.ts

```diff
@@ -24,9 +24,14 @@ const messages = defineMessages({
 })
 export const getTokenFingerprint = (token: Token | DefaultAsset) => {
   const {policyId, assetName} = token.metadata
+  return getAssetFingerprint(policyId, assetName)
+}
+
+export const getAssetFingerprint = (policyId: string, assetName: string) => {
   const assetFingerprint = new AssetFingerprint(Buffer.from(policyId, 'hex'), Buffer.from(assetName, 'hex'))
```

Comment:
There is a new version of this BIP library, can up bump it?, maybe in a PR to base this one